custom-vu/src/main/kotlin/jces1209/vu/JiraDcScenario.kt

```diff
@@ -61,6 +59,13 @@ class JiraDcScenario : Scenario {
                 jira = jira,
                 meter = meter,
                 projectMemory = similarities.projectMemory
+            ),
+            projectIssueNavigatorAction = ProjectIssueNavigatorAction(
+                jira = jira,
+                meter = meter,
+                projectMemory = similarities.projectMemory,
+                projectIssueNavigatorPage = DcProjectIssueNavigatorPage(jira.driver),
```

Comment:
Seems we could reuse `ProjectIssueNavigatorAction` within `ScenarioSimilarities` and pass only the `ProjectIssueNavigatorPage`.

Reply:
they are different